src/Vehicle/FTPManager.h

```diff
@@ -43,7 +43,7 @@ class FTPManager : public QObject
     ///                     a dynamic file creation on the vehicle.
     /// @return true: download has started, false: error, no download
     /// Signals downloadComplete, commandError, commandProgress
-    bool download(const QString& fromURI, const QString& toDir, const QString& fileName="", bool checksize = true);
+    bool download(uint8_t fromCompId, const QString& fromURI, const QString& toDir, const QString& fileName="", bool checksize = true);
```

Comment:
Can you add the param to the API docs as well?

Reply:
ahh ... didn't even thought about that LOL
yes, sure